src/global_alloc_macro/default_alloc.rs

```diff
@@ -20,20 +20,24 @@
 #[macro_export]
 macro_rules! default_alloc {
     () => {
-        $crate::default_alloc!(4 * 1024, 516 * 1024, 64);
+        $crate::default_alloc!({ 4 * 1024 }, { 516 * 1024 }, 64);
     };
     ($fixed_block_heap_size:expr, $heap_size:expr, $min_block_size:expr) => {
-        static mut _BUDDY_HEAP: [u8; $heap_size] = [0u8; $heap_size];
-        static mut _FIXED_BLOCK_HEAP: [u8; $fixed_block_heap_size] = [0u8; $fixed_block_heap_size];
+        #[repr(align(64))]
+        struct _AlignedHeap<const N: usize>([u8; N]);
+
+        static mut _BUDDY_HEAP: _AlignedHeap<$heap_size> = _AlignedHeap([0u8; $heap_size]);
+        static mut _FIXED_BLOCK_HEAP: _AlignedHeap<$fixed_block_heap_size> =
+            _AlignedHeap([0u8; $fixed_block_heap_size]);
 
         #[global_allocator]
         static ALLOC: $crate::buddy_alloc::NonThreadsafeAlloc = unsafe {
             let fast_param = $crate::buddy_alloc::FastAllocParam::new(
```

Comment:
It's suggested to insert a checking function here to make sure all ptr are aligned with 64.

Reply:
If HEAP is aligned to 64 bytes, buddy-alloc allocated ptrs are aligned to 64.

Reply:
Previously, the reported error was misleading and made debugging very difficult. If the error occurs due to incorrect configuration, troubleshooting would be much more straightforward.